### ISSUE TITLE
[bundler] Weird failure to inline of script type=module results in empty inline script with blank src attribute

### ISSUE THREAD

**Issue by usergenic** [Polymer/tools#332]
input:
```
<script type="module" src="../node_modules/@material/mwc-fab/mwc-fab.js"></script>
```
expected:
```
<script type="module">
// rolled-up contents of ../node_modules/@material/mwc-fab/mwc-fab.js
</script>
```
actual:
```
<script type="module" src=""></script>
```

Repro is:
```
git clone https://github.com/material-components/material-components-web-components
cd material-components-web-components
git checkout update-deps
npm install
lerna bootstrap ... something something
touch index.html
polymer build --bundle --entrypoint demos/fab.html
```

**Comment by usergenic** [comment]
First thing to note is I forgot to include `--module-resolution node` in the invocation.  However, this doesn't seem to fix the problem when running `polymer build`.

```
polymer build --bundle --entrypoint demos/fab.html --module-resolution node --npm
```
Basically produces the same output.

However, using polymer-bundler CLI directly doesn't have this problem.  WTFBBQ:
```
polymer-bundler --module-resolution node demos/fab.html --out-dir build --inline-scripts
```
TOTALLY WORKS WTF

**Comment by usergenic** [comment]
oh wait totally wrong it doesnt work.  if you leave off `--inline-scripts` it puts the mwc-fab.js into shared_bundle_2.js and links to that and things are okay.  it still is failing to inline for some reason.

so the good news is this is not a weirdo cli problem but a problem in bundler inlining code.  fixing.

**Comment by usergenic** [comment]
Yay found the cause and fixed it in #334, currently in review.
